src/Shop.jsx

```diff
@@ -26,14 +27,16 @@ const POOLS = {
 
 export default function Shop() {
   const navigate = useNavigate();
+  const addOwnedShopCard = useDeckStore(s => s.addOwnedShopCard);
```

Comment:
I don't actually see any cards when I navigate to the shop. The relics and hexes are different than standard cards. Standard cards are the ones that players will add to their deck. Relics and hexes will be available in a different context that we still need to work out.